build/charts/antrea/conf/antrea-agent-secondary-network.conf

```diff
@@ -0,0 +1,37 @@
+# Name of the primary OpenVSwitch bridge antrea-agent will create for Secondary interface needs.
```

Comment:
"Secondary interface needs" -> "connecting Pod secondary interfaces"?

Reply:
Got it aligned.